rust/debian/Dockerfile

```diff
@@ -39,6 +39,8 @@ RUN --mount=type=secret,id=aws,target=/root/.aws/credentials \
 ## Set up default cargo env vars for cross compiling
 ENV CC_${CROSS_COMPILER_TARGET_ARCH}_unknown_linux_musl=${CROSS_COMPILER_TARGET_ARCH}-linux-musl-gcc \
     CXX_${CROSS_COMPILER_TARGET_ARCH}_unknown_linux_musl=${CROSS_COMPILER_TARGET_ARCH}-linux-musl-g++ \
+    CC_${CROSS_COMPILER_TARGET_ARCH}_unknown_linux_gnu=gcc \
```

Comment:
New version of Ring or CC looks for aarch64-linux-gnu-gcc even when it's running on and aarch64 host... This tells it to just use gcc.